plugins/scaffolder/src/components/fields/EntityPicker/EntityPicker.tsx

```diff
@@ -109,3 +107,70 @@ export const EntityPicker = (props: EntityPickerProps) => {
     </FormControl>
   );
 };
+
+/**
+ * Converts a boolean value `true` to `CATALOG_FILTER_EXISTS`.
+ * Returns the input value if it is not a boolean `true`.
+ *
+ * @param value - The value to convert.
+ * @returns The converted value.
+ */
+function convertTrueToExists(value: string | boolean): string | symbol {
+  if (value === true) {
+    return CATALOG_FILTER_EXISTS;
+  }
```

Comment:
Hmm, I'm a bit hesitant about this, since `true` is a valid value that you might have for a key in an entity. I wonder if there's something else we could go for here that at the same time communicates the intent more clearly. Thinking this kinda stuff:

```yaml
catalogFilter:
  - metadata.annotations.github.com/team-slug: { exists: true }
  - metadata.annotations.github.com/team-slug: __EXISTS__
  - metadata.annotations.github.com/team-slug: { $op: exists }
```

Any thoughts here @freben @backstage/catalog-core ?

Reply:
moved --> https://github.com/backstage/backstage/pull/16779